discovery-provider/src/queries/get_trending_tracks.py

```diff
@@ -55,6 +58,10 @@ def generate_unpopulated_trending_from_mat_views(
     session, genre, time_range, strategy, limit=TRENDING_LIMIT
 ):
 
+    # use all time instead of year for version ML51L
+    if strategy.version == TrendingVersion.ML51L and time_range == "year":
+        time_range = "allTime"
```

Comment:
Might also want to reverse where if `if strategy.version != TrendingVersion.ML51L and time_range == "allTime":` then switch it to year - just in case.